site/docs/v1/tech/apis/tenants.adoc

```diff
@@ -117,6 +118,75 @@ include::docs/v1/tech/apis/_tenant-response-body.adoc[]
 
 include::docs/v1/tech/apis/_tenants-response-body.adoc[]
 
+== Search for Tenants
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+
+This API is used to search for Tenants and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+*Note:* Requests to this API must not include an `X-FusionAuth-TenantId` header, and must be authenticated by an API key which is not scoped to a specific Tenant.
```

Comment:
```suggestion
[NOTE.note]
====
Requests to this API must not include an `X-FusionAuth-TenantId` header, and must be authenticated by a non-tenant scoped API key.
====
```

Reply:
or else what? Does the request fail? Erroneous results?

Reply:
It returns a status of 400, with a body of 
```
{
  "fieldErrors": {},
  "generalErrors": [
    {
      "code": "[restricted]",
      "message": "Your request is not allowed. You cannot search for tenants using a tenant restricted request. Omit the X-FusionAuth-TenantId HTTP header if provided, or repeat the request with an API key that is not assigned to a particular tenant."
    }
  ]
}
```